extensions/file-cluster-service/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.camel.quarkus</groupId>
+        <artifactId>camel-quarkus-extensions</artifactId>
+        <version>3.9.1-SNAPSHOT</version>
```

Comment:
Probably because I am blind & when I reviewed https://github.com/apache/camel-quarkus/commit/84a98bf36e25937d678d60909cb11ee1e27a5566, I did not spot the incorrect version bump.

I'll fix it up after we merge this.

Reply:
Now it fails because of `Caused by: java.lang.RuntimeException: jvmSince 3.10.0 is newer than 3.9.1-SNAPSHOT`, should I revert or wait ? both are ok with me

Reply:
Lets wait for https://github.com/apache/camel-quarkus/pull/5925.

Reply:
@jamesnetherton this should have been fixed